package.json

```diff
@@ -56,6 +56,7 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "fetch-mock": "^4.3.1",
     "file-loader": "^0.8.5",
+    "flexslider": "^2.6.0",
```

Comment:
No, karma is using bower to load this. This is a shit jQuery plugin that has no module system support.